README.md

```diff
@@ -353,7 +353,7 @@ your `.env.local` file. You'll find those secret variables in our shared
 1Password in the "Visualize.admin .env.local" entry.
 
 Make sure to set the `NEXTAUTH_URL` environment variable to
-`http://localhost:3000` and run the dev server with `yarn dev:ssl`.
+`https://localhost:3000` and run the dev server with `yarn dev:ssl`.
 
```

Comment:
```markdown
> Authentication Requirements Checklist
>
> - Create .env.local file
> - NEXTAUTH_URL=https://localhost:3000
> - NEXTAUTH_SECRET=Some Secret
> - Don't forget to import all ADFS_ from `https://vercel.com/ixt/visualization-tool/settings/environment-variables`
```

Maybe we could add something like this?

Reply:
Good call, @noahonyejese. A point that I've wanted to bring up for a while is that I think we currently have a mix of client-specific and non-specific information in the README. It would be great if we could separate the two a bit more consciously and move the information pertaining to the deployment at visualize.admin.ch out of the README and into a dedicated space (e.g. Notion). This would make it more generic – given its context as an open-source project, a good thing imho – and help ensure that potentially security-relevant content (e.g. config) is not inadvertently exposed.

Reply:
Yes, here I would vote for adding these things to our already existing .env.example, which should hopefully make it clear that these values need to be filled in (and provide a centralized place to manage them).

Reply:
https://github.com/visualize-admin/visualization-tool/commit/dc4ed9670f31eda3340ad34ad92955defc1fa56d